js/src/modals/AddAddress/addAddress.js

```diff
@@ -89,7 +89,7 @@ export default class AddAddress extends Component {
         onClick={ this.onAdd }
         ref='addButton'
       />
-    ]);
+    ];
```

Comment:
The `Button`s miss the `key` prop. And while we're at it, we might want to get rid of every `Cancel` button for modals that use the Portal. There is already this big `X` button to cancel/close the modal/Portal.